chia/server/ws_connection.py

```diff
@@ -461,7 +571,13 @@ async def _read_one_message(self) -> Optional[Message]:
             self.bytes_read += len(data)
             self.last_message_time = time.time()
             try:
-                message_type = ProtocolMessageTypes(full_message_loaded.type).name
+                if (
+                    full_message_loaded.type == ProtocolMessageTypes.wrapped_compressed.value
+                    and Capability.CAN_DECOMPRESS_MESSAGES in self.local_capabilities
+                ):
+                    message_type = ProtocolMessageTypes(full_message_loaded.data[0]).name
```

Comment:
you don't seem to be using the protocol message type here either.

Reply:
Same reason as above, will fix.